src/pages/workspaces/workspace/analysis/utils/cost-utils.ts

```diff
@@ -85,16 +92,16 @@ export const runtimeConfigCost = (config: GoogleRuntimeConfig): number => {
   const baseMachinePrice: number = getHourlyCostForMachineType(machineType, computeRegion, false)
 
   const additionalDataprocCost: number = isDataprocConfig(config) ? _.sum([
-    config.numberOfWorkers * getHourlyCostForMachineType(config.workerMachineType, computeRegion, false),
-    getDefaultIfUndefined(config.numberOfPreemptibleWorkers, 0) * getHourlyCostForMachineType(config.workerMachineType, computeRegion, true),
+    config.numberOfWorkers * getHourlyCostForMachineType(config.workerMachineType!, computeRegion, false),
```

Comment:
the worker machine type may/may not be defined(in the type it has a ? after it) this ensures that if there are workers, then the workers machine types are reflected. I've added in a condition now that'll bypass if there are no workers so as to avoid an error with a !'d argument being undefined

Reply:
I've replaced this with a getdefaultifundefined to replace it with defaultDataprocMachineType, let me know if that was a bad idea